app/code/core/Mage/Sales/Model/Order.php

```diff
@@ -1895,8 +1895,10 @@ public function getStatusHistoryCollection($reload = false)
                 ->setOrder('entity_id', 'desc');
 
             if ($this->getId()) {
+                /** @var Mage_Sales_Model_Order_Status_History $status */
                 foreach ($this->_statusHistory as $status) {
```

Comment:
```suggestion
            if ($this->getId()) {
                foreach ($this->_statusHistory as $status) {
```

Reply:
Without the docblock, my VSCode, says it's `mixed`:
![image](https://user-images.githubusercontent.com/1106470/191177031-77e8d58c-5647-4228-939c-37a3390129a3.png)
So how?

Reply:
Then we just keep it.

PhpStorm ...

![pr-2042](https://user-images.githubusercontent.com/5022236/191178149-7dfe07eb-1156-4da1-8090-b53a62ad869b.PNG)